update use tiup client access database (#5028) (#5179)
pingcap/docs: quick-start-with-tidb.md

        {{< copyable "shell-regular" >}}

        ```shell

>         tiup playground v4.0.11 --db 2 --pd 3 --kv 3 --monitor

Reviewer: @King-Dylan @morgo PTAL whether any version-specific change is needed here after 5.0 GA is released.

>         tiup playground v4.0.11 --db 2 --pd 3 --kv 3 --monitor

Reviewer: ```suggestion
        tiup playground v5.0.0 --db 2 --pd 3 --kv 3 --monitor
```

        ```

>         The command downloads a version cluster to the local machine and starts it，such as v5.0.0. `--monitor` means that the monitoring component is also deployed.

Reviewer: ```suggestion
        The command downloads a version cluster to the local machine and starts it, such as v5.0.0. `--monitor` means that the monitoring component is also deployed.
```

>         The command downloads a version cluster to the local machine and starts it，such as v4.0.11. `--monitor` means that the monitoring component is also deployed.

Reviewer: ```suggestion
        The command downloads a version cluster to the local machine and starts it，such as v5.0.0. `--monitor` means that the monitoring component is also deployed.
```

4. Start a new session to access TiDB:

>     1. Use the TiUP client to connect to TiDBs.

Reviewer: ```suggestion
    + Use the TiUP client to connect to TiDBs.
```

        {{< copyable "shell-regular" >}}

        ```shell
        tiup client
        ```

>     2. Also can use the MySQL client to connect to TiDB.

Reviewer: ```suggestion
    + You can also use the MySQL client to connect to TiDB.
```
